pkg/gardener/shoot/converter.go

```diff
@@ -17,12 +17,12 @@ type Converter struct {
 	config    config.ConverterConfig
 }
 
-func NewConverter(config config.ConverterConfig) Converter {
+func NewConverter(config config.ConverterConfig, shootFromState *gardener.Shoot) Converter {
```

Comment:
I'm not convinced to directly pass shoot to the converter. I would introduce some kind of "dynamic configuration". We need to align it with [Audit log PR](https://github.com/kyma-project/infrastructure-manager/pull/472) to have one concistent solution and make rebasing easier.